src/typecode/magic2.py

```diff
@@ -79,18 +79,60 @@
 TYPECODE_LIBMAGIC_DATABASE = 'typecode.libmagic.db'
 
 
+def load_lib_failover():
+  # loader from python-magic
```

Comment:
That's nice extra credits but not needed IMHO since Adam is already credited at the top for the whole file.